src/main/java/core/org/akaza/openclinica/job/XsltTransformJob.java

```diff
@@ -116,7 +116,7 @@
     private static final long KILOBYTE = 1024;
 
     @Override
-    protected void executeInternal(JobExecutionContext context) {
```

Comment:
Making the whole method synchronized could have performance issues since only one thread can run this method at a given time. Can we narrow down the code that is causing this and make that code synchronized?